go-controller/pkg/ovn/gateway_init.go

```diff
@@ -146,10 +146,9 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 		}
 		for _, proto := range enabledProtos {
 			if protoLBMap[proto] == "" {
-				protoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create",
-					"load_balancer",
+				protoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create", "load_balancer",
 					fmt.Sprintf("external_ids:%s_lb_gateway_router=%s", proto, gatewayRouter),
-					fmt.Sprintf("protocol=%s", strings.ToLower(string(proto))))
+					fmt.Sprintf("protocol=%s", strings.ToLower(string(proto))), "options:reject=true")
```

Comment:
You're setting the option regardless of which service controller gets used, but only taking advantage of it in the new service controller

Reply:
yeah, I thought it will be a noop for previous versions ... I need to handle it